src/main/java/seedu/address/logic/parser/TaskCommandsParser.java

```diff
@@ -38,21 +42,37 @@ public TaskCommand parseCommand(String userInput) {
         }
 
         final String commandWord = matcher.group("commandWord");
-        final String arguments = matcher.group("arguments");
+        final String arguments = matcher.group("arguments").trim();
         switch (commandWord) {
 
         case AddTaskCommand.COMMAND_WORD:
             return prepareAdd(arguments);
 
         case DeleteTaskCommand.COMMAND_WORD:
             return prepareDelete(arguments);
+            
+        case FindTaskCommand.COMMAND_WORD:
+            return prepareFind(arguments);
 
```

Comment:
Do we need an additional Clear command? Currently only add and delete clear the filter. Then again - perhaps this should be another issue